.gitignore

```diff
@@ -18,3 +18,5 @@ _[Ss]cripts
 *.dot[Cc]over
 *.pubxml
 *.zip
+/coresamples/CoreCustomSender/project.lock.json

```

Comment:
Should just exclude all `project.lock.json` files - no need to check them in for this repo.